index.js

```diff
@@ -3621,7 +3621,7 @@
   }
   S.props = def('props', {a: [Accessible]}, [$.Array($.String), a, b], props);
 
-  //# get :: Accessible a => (b -> Boolean) -> String -> a -> Maybe c
+  //# get :: Accessible a => (Any -> Boolean) -> String -> a -> Maybe b
   //.
   //. Takes a predicate, a property name, and an object and returns Just the
```

Comment:
This might be hard to articulate without it becoming distracting, but it'd be nice to explain why in this instance a predicate is used instead of relying on a type definition. 

The reason I bring it up is, I've seen quite a few predicate based schema validation libraries.  Very few have their own sophisticated type system like sanctuary-def.  It might be confusing / surprising that a sanctuary function is relying on predicates alone.

*But* I think it does make sense.  It doesn't force a user of sanctuary to understand how to define their own types.  You can use `S.test` to make use of types as predicates.  And functions like `get` are helpful when you aren't completely sure what the input will be, e.g. at the edges of a system.  There are probably more reasons I'm not thinking of.  But I think it's worth while to elaborate on the subject.